Makefile

```diff
@@ -9,21 +9,18 @@
 # `make flowapi-down` to tear down the docker container for flowapi
 # only.
 #
-# By setting the variable FLOWDB_SERVICES you can choose which flowdb
-# version or versions you'd like to use when running `make up`. Examples:
+# By setting the variable DOCKER_SERVICES you can choose which services
+# you'd like to use when running `make up`. Examples:
 #
-#     FLOWDB_SERVICES=flowdb_testdata make up
-#     FLOWDB_SERVICES=flowdb_synthetic_data make up
-#     FLOWDB_SERVICES="flowdb_testdata flowdb_synthetic_data" make up
+#     DOCKER_SERVICES="flowdb_synthetic_data flowapi flowmachine flowauth flowmachine_query_locker" make up
+#     DOCKER_SERVICES="flowdb_synthetic_data" make up
+#     DOCKER_SERVICES="flowdb_testdata flowetl flowetl_db" make up
 #
 # flowmachine and flowapi will connected to the first flowdb service in the list.
```

Comment:
This will fail if there's more than one flowdb service in the list, because it will attempt to allocate them all to the same port. If they were on different localhost ports then it would spin up fine, but I'm not sure which one flowmachine and flowapi would connect to since they'll connect to FlowDB on `flowdb:5432`, which will be the same for all flowdb services. But as @josh-gree mentioned in the issue, it's not clear whether we would ever want to spin up multiple FlowDB services at the same time, so it might be fine to leave things as they are and change this comment to say that the list can contain at most one of "flowdb", "flowdb_testdata" or "flowdb_synthetic_data".

Reply:
Could use some gnu make voodoo to error there using something like.. `$(words $(filter flowdb%, $(DOCKER_SERVICES))` and `$(error..`?

Reply:
Thanks! Following @greenape's suggestion I added an explicit check for multiple flowdb services in the Makefile so that it now throws an error.